misc/services/mobile-webui/mobile-webui-frontend/src/components/ScanHUAndGetQtyComponent.jsx

```diff
@@ -232,15 +252,18 @@ const ScanHUAndGetQtyComponent = ({
     case STATUS_READ_BARCODE: {
       return (
         <>
-          <BarcodeScannerComponent
-            resolveScannedBarcode={handleResolveScannedBarcode}
-            onResolvedResult={onBarcodeScanned}
-            continuousRunning={true}
-          />
+          {useHUScanner ? (
```

Comment:
which is the case when we don't want to use the HU scanner?

Reply:
if I remeber it correctly, the idea was to limit the impact to manufacturing, so all the other places (i.e. picking) where scanHUAndGetQtyComponent is used will not get the new HUScanner